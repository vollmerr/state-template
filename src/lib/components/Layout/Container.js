import React from 'react';

import LayoutBase from './LayoutBase';

const Container = props => (
  <LayoutBase baseClass={'container'} {...props} />
);

export default Container;
