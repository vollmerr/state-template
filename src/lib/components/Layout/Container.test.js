import React from 'react';
import { shallow } from 'enzyme';

import { Container as fromLib } from '../..';
import Container from './Container';

const props = {
  children: 'test-children',
  className: 'test-class',
};

let wrapper;
describe('Container', () => {
  beforeEach(() => {
    wrapper = shallow(<Container {...props} />);
  });

  it('should be exported as part of the library', () => {
    expect(fromLib).toEqual(Container);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
