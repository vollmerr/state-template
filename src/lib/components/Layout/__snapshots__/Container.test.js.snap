// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render correctly 1`] = `
<LayoutBase
  baseClass="container"
  className="test-class"
  tag="div"
>
  test-children
</LayoutBase>
`;
